libs/cart/src/facades/cart/cart.facade.ts

```diff
@@ -81,7 +84,10 @@ export class DaffCartFacade<
 	private _selectCartItemCompositeOptions;
 	private _selectIsCartItemOutOfStock;
 
-  constructor(private store: Store<DaffCartReducersState<T, V, U>>) {
+  constructor(
+    private store: Store<DaffCartReducersState<T, V, U>>,
+    @Inject(DaffCartPaymentMethodIdMap) private paymentMethodMap: Object
```

Comment:
This should be typed as `Dictionary<string>`

Reply:
I don't think we can guarantee that the user will map the methods to strings.

Reply:
That's true, but only because we can't strongly type the injection token. Maybe we should type the actual injection token as `Dictionary<string>` in case someone looks at the type there so that they are encouraged to map to payment ids, even though that typing will not be enforced. I'm just thinking that this feature is pretty complicated and we'll probably need a doc just to explain why this is necessary, so I think it's best to give as many helpful hints as possible. And I think typing the injection token as `Dictionary<string>` instead of `Object` is one such hint. But I agree that here in the facade it should probably just remain an object, since we can't guarantee it yet.

Reply:
hm okay I can retype the token. The only thing that is odd to me is that `Dictionary` is specifically an `@ngrx/entity` type, I wonder if it might be better to instead type it as `{[key: string]: string}`.

Reply:
I think that's fine, since we're dependent on @ngrx/entity anyway, but `{[ket: string]: string}` seems good too

Reply:
changed